.github/actions/cache-package-builds/action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Cache package builds'
+description: 'Cache package builds'
+outputs:
+  cache-hit:
+    description: 'True if cache was hit, false otherwise'
+    value: ${{ steps.package-cache.outputs.cache-hit }}
+runs:
+  using: 'composite'
+  steps:
+    - name: Cache package builds
+      id: package-cache
+      uses: actions/cache@v2
+      with:
+        path: |
+          node_modules/@swarmion-starter/**/dist
+        key: ${{ hashFiles('./contracts/**') }}-${{ hashFiles('./packages/**') }}-${{ hashFiles('./frontend/shared/**') }}
```

Comment:
Isn't this a bit risky to introduce this here ? If the project needs some other pattern for built packages, it may forget to add it here

Reply:
It is yes, but it's the best key I found for this purpose.
We need to recreate the cache every time the source code of a library changes, do you have any idea on how to improve it ?

Reply:
I don't for the moment, maybe we could improve it when using the nx Plugin when generating a new package, but it is better to keep this for later improvements.

Reply:
Can you add a comment here?